pages/index.tsx

```diff
@@ -0,0 +1,7 @@
+import { NextPage } from 'next';
+// @ts-ignore
+import scss from '../page-styles/index.scss'
```

Comment:
why are we `@ts-ignore`ing this?

I'd expect
a) that it should work
b) that it would be a `.module.scss` file
c) that it would be in the same directory as this file

Reply:
I can't place it in the same directory as next freaks out when you place non page files in the page directory

Reply:
at the moment when it is run without TS ignore you get a type error when you open the page

Reply:
I think I have fixed this by including   "@types/node-sass": "^4.11.0", as a dev dependancy

Reply:
lol no

Reply:
much nicer! :)
Thanks

Reply:
in conclusion i am not sure what to do here

Reply:
It is still broken

Reply:
Okay so when you npm run dev it compile, however if you edit the .scss file it will throw a type error

Reply:
try using modules (ie convert the file to `index.module.scss`)

Reply:
I have switched it to .module.scss and that seems to have fixed this